scss/_variables.scss

```diff
@@ -35,6 +35,10 @@ $green-light: #4aa564; // U.S. Web Design Standards color $green-light //
 $green-dark: #287538;
 $error: #cd2026; // U.S. Web Design Standards color (red) $secondary-dark //
 
+// Chart colors
+$dem: #21ADF7;
```

Comment:
Can you set the `$dem` and `$rep` to variables like `$blue-*` or `$red-*` to make them more descriptive and in case they can be used elsewhere?